.github/workflows/try-in-web-ide.yaml

```diff
@@ -0,0 +1,16 @@
+# Add Web IDE link on PRs
+name: web-ide
+
+on:
+  pull_request_target:
+    types: [opened, synchronize]
+
+jobs:
+  add-link:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Web IDE Pull Request Check
+        id: try-in-web-ide
+        uses: redhat-actions/try-in-web-ide@v1
```

Comment:
I'm fine to use @main as well to have always latest new changes

Reply:
FYI for che incubator happy path we publish a next version on every commit
https://github.com/che-incubator/happy-path-tests-action/blob/main/.github/workflows/publish-next.yml